.github/workflows/create-prerelease-branch.yml

```diff
@@ -0,0 +1,109 @@
+# Bump package version numbers and bundle changelogs into changelog file
+name: Create pre-release branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch or tag to create pre-release from'
+        required: true
+        default: 'main'
+      bump_type:
+        description: 'Bump type'
+        required: true
+        type: choice
+        options:
+          - stable-minor
+          - stable-patch
+          - beta
+
+jobs:
+  # create pre-release branch for beta releases
+  create_pre-release:
+    name: Bump versions and make changelog for beta release
```

Comment:
This name is likely wrong - it bumps version for all releases?

Reply:
yes old comment when it was split - has been removed 😊